modulefiles/module_base.hera

```diff
@@ -32,11 +34,14 @@ module load hdf5/1.10.6
 module load esmf/8_1_0_beta_snapshot_27
 module load w3emc/2.7.3
 module load wgrib2/2.0.8
-setenv WGRIB2 wgrib2
+setenv "WGRIB2" "wgrib2"
 
 # Modules not under hpc-stack
 module load hpss/hpss
 module load nco/4.9.1
 module load gempak/7.4.2
 module load ncl/6.5.0
 module load cdo/1.9.5
+module load intelpython/3.6.8
+# netCDF4 is not in the python installation on Hera
```

Comment:
So same thing here, we're moving away from hpc-stack or official modules.  Is this what we want to do?

Reply:
As noted in the commit message, this is definitely a work-around because NetCDF4 is not in the default python build on Hera (it is on WCOSS). We can talk to RDHPCS about having it added, though often their solution is "make your own conda environment", which is pretty much the same problem.

Personally I'd like to have a full python installation/environment in hpc-stack so it is standardized on all machines, but that path is something for @arunchawla-NOAA to determine. Alternatively, just the pyNetCDF4 module could be added there instead of my personal directory.